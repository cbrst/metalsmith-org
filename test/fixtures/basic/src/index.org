* An org-mode Post

With some "amazing", /riveting/, *cooonnntent*.
