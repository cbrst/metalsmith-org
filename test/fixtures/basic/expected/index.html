<h1 id="header-1"><span class="section-number">1</span>An org-mode Post</h1>
<p>With some &#34;amazing&#34;, <i>riveting</i>, <b>cooonnntent</b>.</p>
